From: Joerg Roedel <jroedel@suse.de>
Date: Mon, 3 Sep 2018 11:07:20 +0200
Subject: KVM: VMX: Work around kABI breakage in 'enum vmx_l1d_flush_state'
Patch-mainline: Never, kABI workaround
References: bsc#1106369

We just add a new value to the end of the enum. Hide that
from the kABI checker.

Signed-off-by: Joerg Roedel <jroedel@suse.de>
---
 arch/x86/include/asm/vmx.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/x86/include/asm/vmx.h b/arch/x86/include/asm/vmx.h
index 983f35f9abd8..d41efb49b0f7 100644
--- a/arch/x86/include/asm/vmx.h
+++ b/arch/x86/include/asm/vmx.h
@@ -577,7 +577,9 @@ enum vmx_l1d_flush_state {
 	VMENTER_L1D_FLUSH_COND,
 	VMENTER_L1D_FLUSH_ALWAYS,
 	VMENTER_L1D_FLUSH_EPT_DISABLED,
+#ifndef __GENKSYMS__
 	VMENTER_L1D_FLUSH_NOT_REQUIRED,
+#endif
 };
 
 extern enum vmx_l1d_flush_state l1tf_vmx_mitigation;
-- 
2.12.3

